fdm-app/app/components/custom/combobox.tsx

```
@@ -32,23 +33,20 @@ interface ComboboxProps {
     options: { value: string; label: string }[]
     name: string
     label: ReactNode
+    form: any // TODO: Replace 'any' with a more specific type from react-hook-form if available
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)